List more samples and invite for contribution. (#1697)
google/A2UI: samples/README.md

# A2UI samples and dev tools

This is the registry of examples and dev tools for A2UI.

## Samples

Please, help us maintain samples in this table.
Contribute the fix to the code and/or table below if:

- video is outdated or old
- e2e test is missing
- the example is broken

| Sample                           | Agent  | Renderer | e2e             | Video                    | Issues |
| -------------------------------- | ------ | -------- | --------------- | ------------------------ | ------ |
| [Simple Chat][sc-f-readme]       | Dart   | Flutter  | [yes][sc-f-e2e] | [2026-05-11][sc-f-video] |        |
| [Restaurant finder][rf-a-readme] | Python | Angular  |                 | [2026-06-17][rf-a-video] |        |
| [Restaurant finder][rf-f-readme] | Python | Flutter  | [yes][rf-f-e2e] | [2026-05-02][rf-f-video] |        |
| [Verdure][vr-f-readme]           | Python | Flutter  |                 | [2026-05-11][vr-f-video] |        |
| [Restaurant finder][rf-l-readme] | Python | Lit      |                 | [2026-04-14][rf-l-video] |        |
| [Restaurant finder][rf-r-readme] | Python | React    |                 | [2026-06-17][rf-r-video] |        |

> Have you built and open-sourced a sample using A2UI? Please feel free to add your project [here](../docs/ecosystem/renderers.md).

Reviewer: ```suggestion
Have you built and open-sourced a sample using A2UI? Show it on our [Community Showcase](https://a2ui.org/ecosystem/community/#community-showcase).
```

[vr-f-video]: https://github.com/user-attachments/assets/0a4b1160-0a35-4f1b-ac6d-dd8afb183279
[vr-f-readme]: https://github.com/flutter/genui/blob/main/examples/verdure/README.md

## Dev tools

| Tool                              | Language | Video                        |
| --------------------------------- | -------- | ---------------------------- |
| [Catalog Gallery][gallery-readme] | Flutter  | [2026-06-17][gallery-video]  |
| [Composer][composer-readme]       | Flutter  | [2026-06-17][composer-video] |

[gallery-readme]: https://github.com/flutter/genui/blob/main/dev_tools/catalog_gallery/README.md
[gallery-video]: https://github.com/user-attachments/assets/0cc82b93-5a27-490a-b960-f3fa6ee8280d
[composer-readme]: https://github.com/flutter/genui/blob/main/dev_tools/composer/README.md
[composer-video]: https://github.com/user-attachments/assets/18745f7c-e563-4a1b-80c1-39f1bc3ac606

## Community samples

> Have you built an sample using A2UI? Please feel free to open a pull request to add your project here!

Reviewer: Should this community sample an open-source one?

| Link to sample | Agent | Renderer | Video |
| -------------- | ----- | -------- | ----- |
| ?              | ?     | ?        | ?     |

## Usage in published applications

If you are using A2UI in a published application, please let us know by adding it to this list.

| Link to app | Agent | Renderer | Video |
| ----------- | ----- | -------- | ----- |

> | ?           | ?     | ?        | ?     |

Reviewer: The docsite suggests a different way to showcase samples, see:

> The best way to share your work is to post in [GitHub Discussions](https://github.com/a2ui-project/a2ui/discussions) with a short description, a link to your code, and a demo video (4 minutes or less). Notable projects are featured in the docs and release notes.

https://a2ui.org/ecosystem/community/#community-showcase

Both the docsite and this document should agree somehow? Or have this document link to the docsite?

Author: hm, thank you!

Author: Referenced that doc. Good catch.
